magento/graphql-ce
# Add total_quantity to cart response object

### Description (*)
Provide cart query property for the item summary count without requesting all cart items.

### Fixed Issues (if relevant)
1. magento/graphql-ce#827

### Manual testing scenarios (*)
1. Create empty cart
2. Add products to cart
3. Query cart requesting `total_quantity`
```graphql
{
  cart(cart_id: "{$maskedQuoteId}") {
    total_quantity
  }
}
```

`summary_count` should equal the sum of cart item quantities

### Contribution checklist (*)
